### DISCUSSION TITLE
Multiple instances of Lighthouse

### DISCUSSION THREAD

**Discussion by poupardjea** [GoogleChrome/lighthouse#13727] [General] [answered]
Hello! I am currently working on a project including Lighthouse, this project involves using the Lighthouse foundation to automate audits. I have one instance of Lighthouse running on a server, however a problem appears quickly when we start to perform audits in parallel, indeed the instance does not respond or shows bugs. So I have managed a queue in the api that calls the lighthouse instance but this is a temporary solution.I am looking to deploy multiple lighthouse instances but in order to reduce server load I would like to know how to get the ram/cpu impact of a lighthouse instance?

Thanks for advices !

**Comment by adamraine** [comment, marked as answer]
Running multiple instances of Lighthouse in parallel can lead to performance problems (see https://github.com/GoogleChrome/lighthouse/issues/7187#issuecomment-461838128). You can look into projects like https://github.com/mikestead/lighthouse-batch though.

> I am looking to deploy multiple lighthouse instances but in order to reduce server load I would like to know how to get the ram/cpu impact of a lighthouse instance?

You can track the performance of the process Lighthouse is running in. This is specific to your computer and environment so I can't be of much help here.

**Reply by poupardjea** [reply]
Thank you
